source/template.yaml

```diff
@@ -588,6 +588,7 @@ Resources:
       CompatibleRuntimes:
         - ruby2.7
       RetentionPolicy: Retain
+      CodeUri: aws-ruby-sdk/lib/
```

Comment:
this was definitely not the right way to say "copy the files in this dir to all the others" but I am interested in how to make that happen @crissupb 

The error this results in:

```
    functools.reduce(lambda message, error: message + " " + str(error), e.causes, str(e))
samcli.commands.validate.lib.exceptions.InvalidSamDocumentException: [InvalidResourceException('AWSRubySDKLayer', 'property CodeUri not defined for resource of type AWS::Serverless::LayerVersion')] ('AWSRubySDKLayer', 'property CodeUri not defined for resource of type AWS::Serverless::LayerVersion')
```